packages/react-experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestions.types.ts

```diff
@@ -7,6 +7,7 @@ import {
   IFloatingSuggestionItem,
 } from './FloatingSuggestionsItem/FloatingSuggestionsItem.types';
 import { IRenderFunction, IRefObject } from '@uifabric/utilities';
+import { ISuggestionsControlProps } from '../FloatingSuggestions/Suggestions/Suggestions.types';
```

Comment:
I would just create this interface here rather than depending on floatingsuggestions component